pkg/operator/sync.go

```diff
@@ -34,6 +34,16 @@ import (
 
 const (
 	requiredForUpgradeMachineConfigPoolLabelKey = "operator.machineconfiguration.openshift.io/required-for-upgrade"
+
+	// Platforms for which cloudConfig is available
+	// See https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml#L42
+	platformAWS       = "aws"
+	platformAzure     = "azure"
+	platformBaremetal = "baremetal"
+	platformGCP       = "gcp"
+	platformOpenStack = "openstack"
+	platformVSphere   = "vsphere"
+	platformOvirt     = "ovirt"
```

Comment:
can we reuse the const recently added instead? by doing ToLower

Reply:
Renamed const as per what mentioned at https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L175 . 

Noticed error in [e2e-gcp-op test](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/1658/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1845) which says `Unable to apply 0.0.1-2020-04-17-065403: Unsupported platform GCP`.

But yes, I can look at reusing existing const with doing ToLower. Not sure if it can cause any issue later on.

Reply:
Reusing available platform now

Reply:
no wait, sorry, I should have explained myself better, I'm talking about these: https://github.com/openshift/api/blob/c6e8c9b6eb9ad7f72fc048212a56a1124447c55e/config/v1/types_infrastructure.go#L75

Reply:
ah ok, I understood other way around :D

Reply:
oh this is essentially what I meant with the comment I posted below :)

Reply:
Fixed, hopefully I did right this time. ToLower() is not needed `infra.Status.PlatformStatus.Type` gives  result in the was mentioned in https://github.com/openshift/api/blob/c6e8c9b6eb9ad7f72fc048212a56a1124447c55e/config/v1/types_infrastructure.go#L75